apiary.apib

```diff
@@ -2474,6 +2474,15 @@ Create a new project.
         + title: `New company website` (string, required)
         + description (string, optional)
         + starts_on: `2016-02-04` (string, required)
+        + hourly_rate (object, required)
```

Comment:
Q : Is this what we charge the project's customer? Or is this our internal cost?
Shouldn't we make a clearer difference between those in our namings?

Reply:
Also... this property doesn't really define the rate (only in 1 of the 3 cases...)... it defines something like the _pricing strategy_ or _invoicing method_ or ... so shouldn't we call it like that?
How will this thing be called in the UI?